.kokoro/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Google LLC
+# Copyright 2022 Google LLC
```

Comment:
The copyright year must reflect the year it was introduced, not the most recent edited yet.

```suggestion
# Copyright 2020 Google LLC
```

Reply:
Ah good to know, will revert.